src/components/Radios.tsx

```diff
@@ -0,0 +1,282 @@
+import { Fieldset } from "./Fieldset/Fieldset";
+import PropTypes from "prop-types";
+import React from "react";
+import { Hint, HintProps } from "./Hint";
+import { ErrorMessage, ErrorMessageProps } from "./ErrorMessage";
+import { FormGroup } from "./FormGroup";
+import { Label } from "./Label";
+import { Attributes, DataAttributes } from "../helpers/Attributes";
+
+export interface RadioButton extends React.AriaAttributes, DataAttributes {
+  /**
+   * Specific id attribute for the radio item. If omitted, then `idPrefix` string will be applied.
+   */
+  id?: string;
+  /**
+   * Value for the radio input.
+   */
+  value?: string;
+  /**
+   * Provide attributes and classes to each radio item label.
+   */
+  label?: {
+    id?: string;
+    className?: string;
+    children?: React.ReactNode;
+  };
+  /**
+   * Provide hint to each checkbox item.
+   */
+  hint?: HintProps;
+  /**
+   * Divider text to separate radio items, for example the text "or".
+   */
+  divider?: string;
+  /**
+   * If true, radio will be checked.
+   */
+  checked?: boolean;
+  /**
+   * If true, content provided will be revealed when the item is checked.
+   */
+  conditional?: {
```

Comment:
It doesn't feel like anything's added by making this an object. Can we replace the object with `childrenWhenChecked: React.ReactNode` to be clearer?